### DISCUSSION TITLE
Why Default `suppressHydrationWarning`?

### DISCUSSION THREAD

**Discussion by anaclumos** [shadcn-ui/ui#4565] [General]
@onwn and I recently noticed that there is a default `suppressHydrationWarning` in the RootLayout file. Is there an engineering decision behind this? Why is this needed/included by default?

<img width="821" alt="image" src="https://github.com/user-attachments/assets/7ca60e02-fe22-4d16-a526-38123dfb74bd">

**Comment by HelgiSteinarr** [comment]
It is becuse ThemeProvider from next-themes updates that element. See [https://github.com/pacocoursey/next-themes?tab=readme-ov-file#with-app](https://github.com/pacocoursey/next-themes?tab=readme-ov-file#with-app)

**Reply by anaclumos** [reply]
Amazing, thanks!
